kubeflow/notebooks/notebook.example.yaml

```diff
@@ -0,0 +1,86 @@
+{
+  "apiVersion": "kubeflow.org/v1alpha1",
+  "kind": "Notebook",
+  "metadata": {
+    "annotations": {
+      "accessLocalFs": "false",
+      "disks": "null",
+      "image": "tensorflow-1.10.1-notebook-cpu:v0.3.0",
+      "namespace": "resnet50",
+      "notebookGid": "-1",
+      "notebookPVCMount": "/home/jovyan",
+      "notebookUid": "-1",
+      "owner": "foo",
+      "registry": "gcr.io",
+      "repoName": "kubeflow-images-public",
+      "servicePort": "80",
+      "serviceType": "ClusterIP",
+      "targetPort": 8888,
+      "useJupyterLabAsDefault": true,
+      "ttlSecondsAfterFinished": 300
+    },
+    "name": "notebook",
+    "namespace": "resnet50"
+  },
+  "spec": {
+    "template": {
+      "metadata": {
+        "name": "notebook",
```

Comment:
feels weird that the metadata is redefined here for `name` and `namespace`, what if the user set them to be different values?

Can the pod spec (or job/deployment spec) just inherity the `metadata.labels` and `metadata.annotations` from the CRD?

Reply:
yes, i've removed notebook.example.yaml. The controller does use the notebooks metadata when creating the pod/job/deployment.